core/src/main/antlr4/com/databricks/labs/remorph/parsers/tsql/TSqlParser.g4

```diff
@@ -56,13 +56,16 @@ batchLevelStatement
     | createView
     ;
 
+// TODO: Properly sort out SEMI colons, which have been haphazzardly added in some
```

Comment:
snowflake grammar does it well:
```
batch: sqlCommand (SEMI* sqlCommand)* SEMI*
    ;
```

shall you fix this grammar to follow this pattern as well?

Reply:
Likely so. I cannot see it being different in treatment of batches

Reply:
I will raise another PR tomorrow purely for the SEMI stuff as it will touch a lot of rules. We can get this one in and get the coverage back.

Reply:
@jimidle good. can you also fix the `SEMI` usage that came with https://github.com/databrickslabs/remorph/commit/fed469ed5b97a6d49d72760b5076c39d44fb8e33 for global consistency? :)